auth_backend/templates/user/profile_mentor_info.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'student_base.html' %}
```

Comment:
Is it ok if I delete this file, as we are not using it anymore?

Reply:
yup